tests/test_unit.py

```diff
@@ -0,0 +1,6 @@
+from obal.data.module_utils.obal import get_specfile_sources
```

Comment:
Appears the python 2.7 tests do not like this:

```
  tests/test_unit.py:1: in <module>
      from obal.data.module_utils.obal import get_specfile_sources
  E   ImportError: No module named data.module_utils.obal
```

Is there more I need to add to get it to resolve the path?

Reply:
That did the trick, thanks!